src/test/components/__snapshots__/ZipFileTree.test.js.snap

```diff
@@ -84,7 +84,6 @@ exports[`calltree/ZipFileTree renders a zip file tree 1`] = `
               >
                 <div
                   aria-expanded="true"
-                  aria-label="foo, running time is undefined (undefined), self time is undefined"
```

Comment:
These were incorrect.

Reply:
Can you file a follow-up to check whether we need to implement them for the zip file tree ?